Bottom: 491040ecc8de6610af58d5badece4e5d79150045
Top:    da2f09831f39cac1620a98c833c02d3c356a4c7b
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-18 00:05:15 +1300

Refresh of extended-to-stream

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index 64df1b4..c4f25ac 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -676,14 +676,14 @@ class TestExtendedToStreamDecorator(TestCase):
         self.assertEqual([
             ('startTestRun',),
             ('status',
-             'testtools.tests.test_testresult.TestExtendedToStreamDecorator.test_time',
+             'testtools.tests.test_testresult.TestExtendedToStreamDecorator.test_explicit_time',
              'inprogress',
              None,
              True,
              None,
              now),
             ('status',
-             'testtools.tests.test_testresult.TestExtendedToStreamDecorator.test_time',
+             'testtools.tests.test_testresult.TestExtendedToStreamDecorator.test_explicit_time',
              'success',
               set(),
               True,
